backend/src/main/java/com/benine/backend/database/MySQLDatabase.java

```diff
@@ -107,6 +208,9 @@ public void addPreset(Preset preset) {
       statement = connection.createStatement();
       String sql = preset.createAddSqlQuery();
       statement.executeUpdate(sql);
+      for (String tag : preset.getTags()) {
```

Comment:
I will temporarily remove the foreign key constraint, so I can fix that issue in another pull request. I was already doing too much in this pull request.
With removing the foreign key, the program will still work right.